fullscreen-api/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>Fullscreen API Example</title>
+  <script src="main.js"></script>
+  <link href="main.css" rel="stylesheet">
+</head>
+
+<body>
+  <div id="contents">
+    <p>A quick example for the use of the <code>Fullscreen</code> API. Press <code>ENTER</code> key to toggle Full screen.</p>
+
+    <video src="assets/bigbuckbunny.mp4" id="video" muted autoplay width="300"></video>
```

Comment:
use a more obvious width, like 320 or 480? Also provide the `controls` attribute? Also remove the autoplay attribute — this is generally seen as a bad practice.

Reply:
Also, remove `muted`?